### ISSUE TITLE
Switchbot Hub 2 and Homey - question?

### ISSUE THREAD

**Issue by zdavid1999** [AdyRock/com.switchbot#152]
Hello,

I have a curtain 3 I can’t connect to Homey for some reason, therefore I am considering buying a hub.

I was looking online and found thzt the hub could be used as a remote control for devices such as aircon/heat pump. I looked at the Switchbot app on Homey but the Hub2 is not referenced. I don’t know what a hub is really used for, or how it works. But will I be able to set it up in Homey and therefore automate my Curtains 3 and power on/off my heat pump? That would make buying a hub a better purchase.

thanks for insight…

David

**Comment by AdyRock** [comment]
Yes, the curtains can connect to the hub, and you can control them via the cloud connection. 
You can also add remote-controlled devices using the hub and bring them into Homey.

**Comment by AdyRock** [comment]
You can add the features using the temperature and humidity (hub) driver.

**Comment by NEO-sysdroid** [comment]
Sorry I don't understand how. As the switchBot Hub A(/mini  connects to
many devices with options to manage them e .g. on/off volume up/down,
change input channel. ( I am 80 y so maybe I am not fully ajour)

Den lør. 12. jul. 2025 kl. 18.52 skrev Adrian Rockall <
***@***.***>:
